packages/dds/shared-object-base/src/sharedObject.ts

```diff
@@ -470,7 +479,9 @@ export abstract class SharedObject<TEvent extends ISharedObjectEvents = ISharedO
     constructor(
         id: string,
         runtime: IFluidDataStoreRuntime,
-        attributes: IChannelAttributes) {
+        attributes: IChannelAttributes,
+        private readonly telemetryContextPrefix: string = "",
```

Comment:
I'd consider not to have defaults, forcing people to provide actual values.
If it has default value, I'd assume it should be undefined and mean - please do not collect any data.
I.e. providing empty string is weird, as it will aggregate across DDSs - not sure if this is useful.